vividus-plugin-rest-api/src/main/java/org/vividus/http/HttpTestContext.java

```diff
@@ -25,9 +25,10 @@
 import org.apache.http.client.CookieStore;
 import org.apache.http.client.config.RequestConfig;
 import org.vividus.http.client.HttpResponse;
+import org.vividus.json.JsonContext;
 import org.vividus.testcontext.TestContext;
 
-public class HttpTestContext
+public class HttpTestContext implements JsonContext
```

Comment:
it's better to extract a new class like `JsonTestContext` rather than implement `JsonContext ` by `HttpTestContext `: HTTP. context can be different: it could be JSON, XML, or any other type